internal/config/config.go

```diff
@@ -29,6 +29,11 @@ type Config struct {
 		Namespace struct {
 			Enabled bool `envconfig:"default=true"`
 		}
+		Creator struct {
+			Enabled       bool   `envconfig:"default=true"`
+			RootNamespace string `envconfig:"default=openmfp-system"`
+			FgaGrpcAddr   string `required:"true"`
```

Comment:
i think we can take openfga.openmfp-system.svc.cluster.local:8081 as a default. 

Still we need the possibility to overwrite the value in the helm chart.